scripts/monitoring/cron-send-user-running-pod-count.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+
+COUNT=$(oadm manage-node --list-pods --selector=type=compute | grep Running | wc -l)
```

Comment:
I don't think this is going to work. Based on what we saw with Max's PR yesterday.

I think this needs to be:
```
COUNT=$(KUBECONFIG=/etc/openshift/master/admin.kubeconfig oadm manage-node --list-pods --selector=type=compute | grep Running | wc -l  ; exit ${PIPESTATUS[0]})
```


The KUBECONFIG will make sure we have the correct auth, the PIPESTATUS will make sure we exit with the correct exit code.